recipe/meta.yaml

```diff
@@ -1,36 +1,39 @@
-{% set version = "3.4.1" %}
+{% set version = "3.4.3" %}
 {% set blas_variant = "openblas" %}
 
 package:
   name: opencv
   version: {{ version }}
 
 source:
-  fn: opencv-{{ version }}.tar.gz
-  url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
-  sha256: f1b87684d75496a1054405ae3ee0b6573acaf3dad39eaf4f1d66fdd7e03dc852
+  - url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
+    fn: opencv-{{ version }}.tar.gz
+    sha256: 4eef85759d5450b183459ff216b4c0fa43e87a4f6aa92c8af649f89336f002ec
+    patches:
+      # https://github.com/opencv/opencv/pull/12465
+      - 12465-bayer2rgba.patch
```

Comment:
Normally we don't maintain patches at this feedstock as there are usually many that people want to add and the maintenance burden of adding these is quite high. Should add this is not true of all feedstocks, but has been historically true of the OpenCV feedstock in particular. IOW the builds take a while, the builds frequently fail, maintainers are on the hook for these patches as contributors don't normally maintain them, etc..

That said, if this is particularly important, we can let it slide. It would just be good not to be accumulating a bunch more patches for OpenCV going forward. Instead we should encourage them to make a new release so we can integrate all upstreamed changes as part of the release process.

Reply:
I get it. Ill take it off

Do you mind reviewing this pr so that I can upload the package to my own channel?

https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/21